docs/master/api-reference/directives.md

```diff
@@ -300,6 +446,19 @@ type Query {
 }
 ```
 
+### Definition
+
+```graphql
+directive @complexity(
+  """
+  Provide your own function to calculate complexity.
+  """
+  resolver: String
```

Comment:
Let's add a quick explanation of the `Class@method` format here and in other places where we require such method references.

Reply:
I changed it to:

```
Provide your own function to calculate complexity.
Specify a fully qualified class name (FQCN) and method.
```

There is an example right below it. That should be sufficient. :-)